xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-webjar/src/main/webjar/layouts/livedata-table.html

```diff
@@ -0,0 +1,90 @@
+<template>
+    <table class="xwiki-livetable">
+
+        <thead>
+
+            <tr>
+
+                <th
+                    v-for="(col, cIdx) in cols"
```

Comment:
I'm not a fan of cryptic variable names. I don't think reducing the variable name by only **2** characters is worth in this case.

Reply:
I think that `col` is pretty understandable, I changed `cIdx` to `colIdx` if that sounds better, if though it is only used in the next line just to maintain the v-for index

Reply:
I was only referring to ``colIdx``. I would have used ``colIndex``, which is only 2 characters longer. For me it makes sense to use abbreviation when the win is significant, e.g. i18n vs. internationalization. And if you really want to be concise then you can simply use ``i`` as this is a local short-lived variable.